lib/ansiblelint/cli.py

```diff
@@ -110,17 +110,12 @@ def get_cli_parser() -> argparse.ArgumentParser:
                         help="parseable output including severity of rule")
     parser.add_argument('-r', action=AbspathArgAction, dest='rulesdir',
                         default=[], type=Path,
-                        help="specify one or more rules directories using "
-                             "one or more -r arguments. Any -r flags override "
-                             "the default rules in %s, unless -R is also used."
```

Comment:
I don't like this being removed. Why can't the var just be in some utils module instead?

Reply:
@webknjaz If you really want we could follow Ansible model and create a constants module where to put bits like this (also exit codes).

Regarding printing the internal include path with --help, i have some doubts that this is really needed. If I remember well, I was the one that added this but the reality is that there is not much use to it.

Use of logging provides a far better feedback for those wanting to understand the internals.

Reply:
This is not the same as logging. More comprehensive help output improves the UX of folks who are trying to figure out what CLI args to use even before running the thing.

Reply:
I added it back to help, but without mentioning it twice. Help needed rephrasing anyway. I also created constants module, I will move other constants to it in follow-ups.